config/secrets.yml

```diff
@@ -30,3 +36,6 @@ test:
 
 production:
   secret_key_base: <%= ENV["SECRET_KEY_BASE"] %>
+  twilio_account_sid:  <%= ENV.fetch("SECRET_KEY_BASE", "AC7f2af091f4a2bbed0a98517827216672") %>
+  twilio_auth_token:  <%= ENV.fetch("SECRET_KEY_BASE", "19b4bcbe3b1a6cca4ece4efda236fcd9") %>
```

Comment:
But, won't that cause the same problem that we had before, exposing our token?

Reply:
gfdi